bacon/commons/Decimal+toFormattedString.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Decimal+toFormattedString.swift
+//  bacon
+//
+//  Created by Lizhi Zhang on 3/4/19.
+//  Copyright © 2019 nus.CS3217. All rights reserved.
+//
+
+import Foundation
+
+extension Decimal {
+    var toFormattedString: String? {
```

Comment:
Should this be a method instead, and all the configuration below be accepted as arguments? That'll make it more reusable.

Reply:
Oh it was because I wouldn't want to allow any arguments. It is to standardise formatting of the decimal numbers throughout the app in different pages.